README.md

```diff
@@ -6,7 +6,7 @@
 [![Nuget version](https://img.shields.io/nuget/v/MudBlazor.ThemeManager?color=ff4081&label=nuget%20version&logo=nuget&style=flat-square)](https://www.nuget.org/packages/MudBlazor.ThemeManager/)
 [![Discord](https://img.shields.io/discord/786656789310865418?color=%237289da&label=Discord&logo=discord&logoColor=%237289da&style=flat-square)](https://discord.gg/mudblazor)
 
-Blazor Theme Manager component for MudBlazor library. Can be used live or during development to fast and easy try out different theme settings.
+Blazor Theme Manager component for MudBlazor library.  Can use either live or during development &mdash; fast and easy to try out different theme settings!
```

Comment:
Actually your suggestion doesn't seem to be good grammar either to be honest ;)

I would change to `Can be used in production to allow customization or during development to try out different theme settings quickly and easily.`

Reply:
Agreed. The first part still bugs me too.  So are you saying the PROD is fully customizable?  If so, how about:

Fully customizable for production usage or during development to experiment with different theme settings quickly and easily.